src/ML-POD/pair_pod.h

```diff
@@ -0,0 +1,125 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/ Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef PAIR_CLASS
+// clang-format off
+PairStyle(mlpod,PairPOD);
+// clang-format on
+#else
+
+#ifndef LMP_PAIR_POD_H
+#define LMP_PAIR_POD_H
+
+#include "pair.h"
+
+namespace LAMMPS_NS {
+
+  class PairPOD : public Pair {
+  public:
+      PairPOD(class LAMMPS *);
+      ~PairPOD() override;
+      void compute(int, int) override;
+
+    void settings(int, char **) override;
+    void coeff(int, char **) override;
+    void init_style() override;
+    double init_one(int, int) override;
+    double memory_usage() override;
+    void *extract(const char *, int &) override;
+
+    int dim = 3;
```

Comment:
Please move all initializations from the header files to either initializer lists of the constructor (for pointers) or explicit statements in the constructor.

Reply:
Moved  initializations to the constructor.